src/lib/device-config.ts

```diff
@@ -80,9 +79,6 @@ export const generateConfig = async (
 			user,
 			apiKey,
 			pubnub: {},
-			mixpanel: {
```

Comment:
@CameronDiver is there any change that this could break older supervisors?

Reply:
Discussing in FD:
https://www.flowdock.com/app/rulemotion/r-supervisor/threads/OGBukk82q2QrkMoA-M9f33HrP_1